### DISCUSSION TITLE
notification not marked as done not shown, remains hidden

### DISCUSSION THREAD

**Discussion by sezero** [community/community#126756] [Projects and Issues]
### Select Topic Area

Question

### Body

Some notifications not marked as 'done' are not shown and stay hidden:

If any of the affected issues or PRs are updated, they still won't show on the list but the blue dot remains, at which time I have to hunt and find which issue or PR it is (or check my email.)

For me, this issue started only today. Has anyone else run into this?

![bad-notify](https://github.com/community/community/assets/4222725/85672f20-caff-4230-b5da-ee8d61a65c14)

**Comment by iTechTop** [comment]
🤔It sounds like you’re experiencing an issue with GitHub notifications not updating correctly. This is a known issue that can occasionally happen. Here are some steps you can take to troubleshoot and possibly resolve the problem:

 1. Check GitHub Status
Sometimes, GitHub services might be experiencing issues. Check the [GitHub Status](https://www.githubstatus.com/) page to see if there are any ongoing incidents that might be affecting notifications.

 2. Clear Browser Cache and Cookies
Clearing your browser's cache and cookies can help resolve display issues:

- Chrome: `Settings > Privacy and security > Clear browsing data`
- Firefox: `Settings > Privacy & Security > Cookies and Site Data > Clear Data`
- Safari: `Preferences > Privacy > Manage Website Data`

 3. Log Out and Log Back In
Logging out of GitHub and then logging back in can sometimes refresh your session and resolve issues with notifications.

 4. Try a Different Browser
Try accessing your GitHub notifications using a different browser to see if the issue persists. This can help determine if the problem is browser-specific.

 5. Check Notification Settings
Ensure your notification settings are correctly configured:

1. Go to your Profile picture > Settings.
2. Select Notifications.
3. Review your notification settings for any anomalies.

 6. Use GitHub Mobile or Desktop Apps
Try using the GitHub mobile app or the GitHub Desktop app to see if notifications are correctly displayed there.

 7. Email Notifications
Since you mentioned email notifications, you can use these as a temporary workaround to stay updated on issue and PR updates.

 8. Report the Issue
If none of the above steps resolve the issue, consider reporting it to GitHub Support. Provide them with details about the problem, including when it started and any steps you’ve already taken to try and resolve it.

I hope this is helpful for solving the issue.

**Comment by sezero** [comment]
Thanks @monolab825: Using firefox on linux (and windows.) I logged out, cleared all browser history / cookies / etc, re-logged in. Still the same. Logged in using MS Edge on a windows os, still the same. Github status page lists everything as green. My notification settings show nothing unusual. So I assume something wrong at github itself.

How do I report this to github? (Don't know the procedure.)

**Comment by sezero** [comment]
It seems to have resolved magically just now. Closing.
